src/main/resources/db/changesets/20231129_snapshotrequests.yaml

```
@@ -0,0 +1,60 @@
+databaseChangeLog:
+  - changeSet:
+      id: snapshot_requests
+      author: rjohanek
+      changes:
+        - createTable:
+            tableName: snapshot_requests
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: dataset_id
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_dataset_snapshot_requests
+                    references: dataset(id)
+                    deleteCascade: true
+              - column:
+                  name: request_name
+                  type: text
+                  constraints:
+                    nullable: false
+              - column:
+                  name: request_research_purpose
+                  type: text
+                  constraints:
+                    nullable: false
+              - column:
+                  name: snapshot_builder_request
+                  type: jsonb
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_email
+                  type: text
+                  constraints:
+                    nullable: false
+              - column:
+                  name: create_date
```

I think this is usually `created_date`
```suggestion
                  name: created_date
```